### PR TITLE
Add allocation updates to interaction_events

### PR DESCRIPTION
**Pull request by crabsinger**
Add allocation updates to interaction_events

### CODE REVIEW

File: api/hasura/actions/_handlers/updateAllocations.ts [hunk 1]
```diff
@@ -133,6 +133,22 @@ async function handler(req: VercelRequest, res: VercelResponse) {
     const existingGift = pending_sent_gifts.find(
       g => g.recipient_id === gift.recipient_id
     );
+
+    ops['insertInteractionEvents'] = {
```
Comment (crabsinger) [inline api/hasura/actions/_handlers/updateAllocations.ts:137]: more verbose to add it here, but its wrapped into one sql mutation vs if we used the insertInteractionEvents call...

**Review by topocount — DISMISSED** [review]
This is great!

One thing to just verify: if any part of a tx fails, every single mutation would be rolled back. If there is no risk of erroneous rollback, this is good to go.

**Comment by topocount** [comment]
> @topocount good to know, a mutation is always wrapped in a TRANSACTION from a sql standpoint?

@crabsinger when there is more than one mutation in a request, hasura wraps it in a database tx: https://hasura.io/docs/latest/mutations/postgres/multiple-mutations/#execution

**Comment by levity** [comment]
why was this one closed?

**Comment by crabsinger** [comment]
> why was this one closed?
@levity this set of changes got batched together into this PR: https://github.com/coordinape/coordinape/pull/1357

**Pull request closed without merging**